unreleased/mysql-add-endpoint-attribute.yaml

```diff
@@ -0,0 +1,5 @@
+change_type: enhancement
```

Comment:
Technically, I think we need to call this a breaking change, as it may break data continuity for some consumers.